repos.d/gentoo/guru.yaml

```diff
@@ -0,0 +1,37 @@
+###########################################################################
+# GURU
+###########################################################################
+- name: guru
+  sortname: gentoo~guru  # place right after Gentoo
+  type: repository
+  desc: GURU
+  statsgroup: Gentoo
+  family: gentoo
+  ruleset: [ gentoo, guru ]
+  color: '62548f'
+  update_period: 12h
+  minpackages: 100
+  default_maintainer: guru@gentoo.org
+  sources:
+    - name: guru
+      fetcher: GitFetcher
+      parser: GentooGitParser
+      url: https://github.com/gentoo-mirror/guru.git
+      require_md5cache_metadata: true
+      require_xml_metadata: false
+      sparse_checkout: [ '**/*.ebuild', '**/metadata.xml', 'metadata/md5-cache/*' ]
+  repolinks:
```

Comment:
Please avoid link spam and use simpler desc's. `Gentoo wiki page for Project:GURU`, `Git repository`, `Git repository mirror on GutHub`. Not sure if article is needed. Also these desc's are broken YAML.

Reply:
yes, I got excited here with number of urls